src/styles/global.css

```
@@ -278,3 +281,29 @@ html {
+    overflow: hidden;
+
+      .overlay{
+          display: block;
+      }
+ }
+
+
+dialog {
+    scale: 0;
+
+    transition: scale .4s ease, display .4s ease allow-discrete;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)